__tests__/app.test.js

```diff
@@ -61,6 +61,7 @@ describe("/api/articles", () => {
                   topic: expect.any(String),
                   created_at: expect.any(String),
                   votes: expect.any(Number),
+                  comment_count: expect.any(Number),
```

Comment:
please don't amend old tests. You should be adding new tests for new functionality